src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/OverlappingEdgeCheck.java

```diff
@@ -182,13 +182,13 @@ private boolean isPartOfClosedWay(final Edge object)
         // Loop through out going edges with the same OSM id
         while (nextEdge != null)
         {
-            wayIds.add(nextEdge.getIdentifier());
+            wayIds.add(nextEdge.getOsmIdentifier());
```

Comment:
It looks like the items in `wayIds` should be atlas identifiers, otherwise the search will return true for all multi edge ways.

Reply:
Agreed. Will change it.